classes/components/listPanels/PKPSelectReviewerListPanel.inc.php

```diff
@@ -108,6 +111,12 @@ public function getConfig()
                 'max' => 75,
                 'valueLabel' => __('common.lessThan'),
             ],
+            [
```

Comment:
This filter should only appear when we're on round 2 or later. So when this is the first round, the filter shouldn't exist.